### ISSUE TITLE
misprint: circuits.core.components:BaseComponent

### ISSUE THREAD

**Issue by byrgazov** [circuits/circuits#243]
```
# circuits.core.components:BaseComponent.__init__:
# ...
# TODO: Document this feature. See Issue #88
if v is not self and isinstance(v, BaseComponent) \
        and v not in ('parent', 'root'):
# ...
```

**Comment by spaceone** [comment]
@byrgazov Where is the typo? I can't see it.

**Comment by byrgazov** [comment]
I don't remember what the problem was :-).
Maybe it should be: `k not in ('parent', 'root')`
The second part of the expression makes no sense: `isinstance(v, BaseComponent) and v not in ('string1', string2', ...)`

**Comment by spaceone** [comment]
Fixed via 5e1ae1c
